### PR TITLE
PIL-1244-Added hitSlop to the swap icon

### PR DESCRIPTION
**Pull request by Yash-Chaubal**

### CODE REVIEW

File: src/screens/Exchange/Exchange/Exchange.js [hunk 1]
```diff
@@ -245,6 +245,8 @@ const TouchableSwapIcon = styled.TouchableOpacity`
   align-self: center;
 `;
 
+const swapIconHitSlop = { top: 20, bottom: 20, left: 50, right: 50 };
```
Comment (IAmKio) [inline src/screens/Exchange/Exchange/Exchange.js:248]: Could you check if we have something already for this? Have a look in `common.js` for `hitSlop20`

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just one minor thing, cheers!

**Review by IAmKio — APPROVED** [review]
LGTM! 🙏

File: src/screens/Exchange/Exchange/Exchange.js [hunk 2]
```diff
@@ -66,6 +66,7 @@ import FromAssetSelector from './FromAssetSelector';
 import ToAssetSelector from './ToAssetSelector';
 import OfferCard from './OfferCard';
 import { useFromAssets, useToAssets, useOffersQuery, sortOffers } from './utils';
+import { hitSlop50w20h } from '../../../utils/common';
```
Comment (divyaBacancy) [inline src/screens/Exchange/Exchange/Exchange.js:69]: Please import it in under Utils

```suggestion
// Utils
import { hitSlop50w20h } from 'utils/common';
```

File: src/utils/common.js [hunk 3]
```diff
@@ -569,6 +569,14 @@ export const hitSlop20 = {
...
 };
 
+export const hitSlop50w20h = {
+  top: 20,
+  bottom: 20,
+  left: 50,
+  right: 50,
+};
+
+
```
Comment (divyaBacancy) [inline src/utils/common.js:579]: remove the extra line.
```suggestion
```

**Review by divyaBacancy — COMMENTED** [review]
just code formatting changes

File: src/screens/Exchange/Exchange/Exchange.js (hunk 2 above)
Comment (Yash-Chaubal) [reply src/screens/Exchange/Exchange/Exchange.js:69]: Ah my apologies, I used the auto import function in visual studio code

**Pull request merged**